VolleyManagement.UI/spa/environments/environment.integrated.ts

```diff
@@ -0,0 +1,11 @@
+// The file contents for the current environment will overwrite these during build.
+// The build system defaults to the dev environment which uses `environment.ts`, but if you do
+// `ng build --env=prod` then `environment.prod.ts` will be used instead.
+// The list of which env maps to which file can be found in `.angular-cli.json`.
+
+export const environment = {
```

Comment:
I think we need one more file so we will have dev and prod build for VOU integration

Also can we name variable vouIntegration?

Reply:
1. No need for this, at least now. Developer simply set environment.integrated to "true". And everything works. You should remember, we don't use specific flag with environment when running "ng serve".
2. I don't know if it possible, maybe for someone who really good in webpack and angular cli configuration, because you want config which will use files with variable environment in one case and with variable vouIntegration in other. I can simply rename field inside environment from "integrated" to  "vouIntegration" if you really think that it is needed))